examples/webpack.config.js

```diff
@@ -74,21 +71,4 @@ const config = {
 
 !PROD && (config.devtool = 'source-map');
 
-PROD && config.plugins.push(
-     new webpack.optimize.UglifyJsPlugin({
-        compressor: {
-            warnings: false,
-        }
-    })
-);
-
-PROD && config.plugins.push(
-    new webpack.DefinePlugin({
-        'process.env': {
-            'NODE_ENV': JSON.stringify('production')
-        }
-    }),
-    new MinifyPlugin()
-);
```

Comment:
except `MinifyPlugin` that I removed, this is all defaulted by `mode="production"`.